src/CareerBuilder/OAuth2/Flows/FlowClientFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace CareerBuilder\OAuth2\Flows;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\MessageFormatter;
+use GuzzleHttp\Middleware;
+
+class FlowClientFactory
+{
+    public static function create($configs, $logger)
+    {
+        $stack = HandlerStack::create();
+        $stack->push(
+            Middleware::log(
+                $logger,
+                new MessageFormatter('{req_body} - {res_body}')
+            ),
+            'OAuthClientLogger'
+        );
+
+        return new Client([
+            'base_uri' => $configs['base_url'],
+            'handler' => $stack,
+        ]);
+    }
+}
```

Comment:
❌Hey, this code is not following the PSR-12 standard, which we are trying to strongly follow in this codebase.
Doc: https://www.php-fig.org/psr/psr-12